build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.2.0'
+	id 'io.spring.dependency-management' version '1.1.4'
+}
+
+group = 'com.github.arkadiusz97.sms_filter_service'
```

Comment:
W Javie wykorzystuje się camelCase, jednak Oracle przewidział wyjątek dla nazw package i zastosowałem się do ich konwencji: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
`Package names are written in all lower case to avoid conflict with the names of classes or interfaces.`